.github/workflows/package-build.yml

```diff
@@ -98,6 +103,8 @@ jobs:
           cmd: |
             if [ "${{ inputs.machine-config }}" = "true" ]; then
               cmake ../src -DCMAKE_BUILD_TYPE=${{ env.BUILD_TYPE }}
+            elif [ "${{ inputs.build-fuzzers }}" = "true" ]; then
+              cmake ../src -DCMAKE_BUILD_TYPE=Debug -DCMAKE_C_COMPILER="/usr/bin/clang" -DCMAKE_CXX_COMPILER="/usr/bin/clang++" -DTWEAK_VERSION=${{ steps.version.outputs.tweak }} -Duse_prov_client=ON -Dhsm_type_symm_key=ON -DCOMPILE_WITH_STRICTNESS=ON -DBUILD_TESTS=OFF -DBUILD_MODULETEST=ON -DBUILD_SAMPLES=OFF -Duse_default_uuid=ON -DBUILD_ADAPTERS=ON -DBUILD_FUZZER=ON
```

Comment:
Can we also enable `BUILD_TESTS`?

Reply:
Or, can we also exclude BUILD_MODULETEST (why would we need this one for fuzzing)?

Reply:
Will disable BUILD_MODULETEST.